### ISSUE TITLE
Add hourly demand chart for hybrid heat pump on hydrogen

### ISSUE THREAD

**Issue by AlexanderWirtz** [quintel/etmodel#3379]
Martijn Douwes brought this to my attention for the International Scenario. This seems like a bug, no?

![image](https://user-images.githubusercontent.com/308563/84283322-24f9eb00-ab3b-11ea-9053-65e8154f1c30.png)

Please re-assign at will and if this is not part of the deploy remove the milestone as you see fit.

**Comment by marliekeverweij** [comment]
Good point. We made the decision to leave hydrogen out of this chart, but I do not know why anymore. Let's add it!

**Comment by marliekeverweij** [comment]
I've tried adding the series, but it gets a bit messy all in one chart! 😝 
![Screenshot 2020-06-15 at 17 34 46](https://user-images.githubusercontent.com/32833996/84677094-84d30600-af2e-11ea-9663-4d87e980b00f.png)

Proposal: add a separate chart for the HHP hydrogen! However, that's too much work before the deploy tomorrow, so have added this todo to the next deploy milestone

Series and translations can be found here:
https://github.com/quintel/etmodel/tree/hhp-chart
https://github.com/quintel/etsource/tree/hhp-chart

**Comment by marliekeverweij** [comment]
Do we want to add this chart in this deploy? @ChaelKruip or anybody else who is in favour or against!
